src/app/clusters/scenes-server/SceneTableImpl.h

```diff
@@ -40,8 +40,7 @@ using clusterId = chip::ClusterId;
 class DefaultSceneHandlerImpl : public scenes::SceneHandler
 {
 public:
-    static constexpr uint8_t kMaxValueSize = 4;
-    static constexpr uint8_t kMaxAvPair    = 15;
+    static constexpr uint8_t kMaxAvPair = 15;
```

Comment:
Nit, if this number is present in the spec, please add the reference in comments.

Reply:
It's not in the spec, what about a CHIP_CONFIG?

Reply:
that could do.